src/chromo/cli.py

```diff
@@ -328,6 +344,14 @@ def main():
     task_id = None
     try:
         model = args.model(evt_kin, seed=args.seed)
+        if args.config:
```

Comment:
At first glance it is confusing to use arg.config for config file name and then for the content of the file.

Reply:
Yes, I was lazy and reused the same attribute. Do you have a suggestion to make it more clear? Perhaps with a comment?

Reply:
I now write the code to `args.config_code`. Is that better?

Reply:
I am not sure if I like it better with args.config_code. Both `args.config` and `args.config_code` store a `str`, so even if you think in static types, it is fair reuse. Perhaps a comment is sufficient to declare that args.config is overridden with the code.

Reply:
I now return `(args, configuration)` from `parse_arguments`, where `configuration` contains the optional configuration code. `args.config` remains untouched and is now used to show the configuration file in the CLI banner.